library/bigip_selfip.py

```diff
@@ -86,7 +86,19 @@
       validate_certs: "no"
       vlan: "vlan1"
   delegate_to: localhost
-
+- name: Create Self IP with a Route Domain
+  bigip_selfip:
+      server: "lb.mydomain.com"
+      user: "admin"
+      password: "secret"
+      validate_certs: "no"
+      name: "self1"
+      address: "10.10.10.10"
+      netmask: "255.255.255.0"
+      vlan: "vlan1"
+      rd: "10"
+      allow_service: "default"
+  delegate_to: localhost
```

Comment:
I didn't see any spaces between the other examples, let me know if you still think there should be spaces between the examples...